src/container/Displayslots.js

```diff
@@ -1,13 +1,16 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
+import {Col , Row} from 'reactstrap'
 import "./Home.css";
-// import Button from "../components/Button.js";
 import Header from "../components/Header.js";
 import Footer from "../components/Footer.js";
 import { NavLink} from 'react-router-dom';
 import Image from "../components/Image.js";
-import { Container, Card, CardTitle, CardText, Row, Col, Button } from 'reactstrap';
-import moment from 'moment';
+import moment from 'moment'
+import { Dropdown, DropdownToggle, DropdownMenu, DropdownItem, Button } from 'reactstrap';
```

Comment:
Line 3 is importing from reactstrap too, should these be combined for clarity?

Reply:
@laurieainley okay i will combine them